src/components/SourceTabs.js

```diff
@@ -142,7 +142,7 @@ const SourceTabs = React.createClass({
       label: "Show source",
       accesskey: "s",
       disabled: false,
-      click: () => showSource(tab)
+      click: () => showSource(tab, Math.random())
```

Comment:
hmm, this seems strange. 

Perhaps we need to change the way Source Tree checks in receiveProps, and always focus or something like that